### ISSUE TITLE
Troubleshoot CREATE_GARDEN_MUTATION

### ISSUE THREAD

**Issue by capndave** [capndave/gardenbuilder-frontend#27]
## Need

Please:

- [ ] Look at `AddGarden.js` and make sure that the results of the CREATE_GARDEN_MUTATION are pushed to the Apollo cache appropriately.
- [ ] Ideally, they will be pushed to the same object in the cache that is saved after running the GET_USER_GARDENS query in `GardenList.js`.
- [ ] Please ensure that the `GardenList` view also updates after the `CREATE_GARDEN_MUTATION`.

## Notes

#26 needs to be completed before starting this

## Extra Credit

If you'd like to test that the GardenList updates with the newly-created garden that would be awesome!

**Comment by paytonagreen** [comment]
I'm not sure why, but this seems to be fully functional now?

**Comment by paytonagreen** [comment]
I've tested that AddGarden calls the mutation correctly and that GardenList renders the results of the query correctly. The view does update in the browser, as well. Unsure how to make an integration test work without pushing/pulling actual data to/from the server every time the test runs.

**Comment by capndave** [comment]
I imagine it's possible with the right kind of mock response, but might require separating the query call into its own module or otherwise relocating/separating some logic. I don't think it's super high priority. We can always make a new issue just to make this integration test and someone who's more passionate/informed about this can take it on if they want.
